### PR TITLE
PREOPS-1335: Phil's edits on ES milestones, commissioning observations, DP1/DP2/DR1 contents, impact on survey cadence

### PR DESCRIPTION
**Pull request by drphilmarshall**
This PR will track my content changes to RTN-011 for ticket PREOPS-1335. It will support review by (primarily) @leannep before we merge my edits into the primary ticket branch (which will be reviewed by many others). I'll add Leanne as a reviewer when the branch is ready for her review. 

PS. Note that I had to drop the hyphen from the branch name in order to be able to create the new branch... I can look at renaming the main ticket branch. _Update: it's not possible, since the git rename command to remove the hyphen _also_ doesn't work because the branch name has a hyphen in it... Lesson learned, don't make branch names with hyphens in them._

### CODE REVIEW

**Comment by drphilmarshall** [comment]
@leannep My edits to RTN-011 are ready for your review! Hopefully you can merge, rebase, and fix any problems in your branch - feel free to merge this PR if you agree. Otherwise, I'll address any comments you leave in a PR review. Thanks!

File: commissioning.tex [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
+Commissioning data collection is planned to take place in phases.
+A period of on-sky engineering time will be followed by a set of observations designed to help optimize the system, before the \svs are carried out.
+The SV surveys are designed to support scientific analyses that validate the system's performance, and allow Rubin to demonstrate operations readiness.
 
-Text introducing figure.
-
-Key components of SO and SV phases with each Camera. 20 year depth tests. 1000 sq deg pilot survey.
+The figure indicates a number of key components of the system optimization and SV phases planned for each camera.
+These include 20 year depth tests in select fields, and a 1000 sq.\ deg.\, multi-band, wide-fast-deep (WFD) 1-2 year depth ``pilot'' survey.
+Field selection will be carried out by the Commissioning Team, taking into account a wide variety of constraints as well as a ``menu'' of science opportunities provided by the LSST Science Community.
```
Comment (leannep) [inline commissioning.tex:21]: The community alone will not decide the menu. I'd rephrase as 

Field selection will be carried out by the Commissioning Team, taking into account a wide variety of constraints  as well as a ``menu'' of science opportunities to which the LSST Science Community may contribute.

File: timeline.tex [hunk 2]
```diff
@@ -1,24 +1,48 @@
...
+The date ranges in the Table are derived from the ``Celebratory Milestones'' list published at monthly intervals on the Rubin Project website;\footnote{\url{https://www.lsst.org/about/project-status}} shown here are the milestones as of October 2022, which use Project controls data from August 2022.
+
+Milestone dates are given as min-max ranges to indicate their uncertainty.
+Typically the near date corresponds to the current Project forecast, plus any additional operational uncertainty.
+The late date corresponds (approximately) to the current Project ``late date'' plus any additional operational uncertainty: this late date cannot be surpassed without the Project rebaselining its schedule.
+
+The late dates for the DP2 and DR1 data release milestones allow for the possibility that the Project completes within its late date, but reduces the amount of on-sky LSST Cam commissioning time in order to do so.
+In this eventuality, the operations team would spend up to 3 months at the start of the full/survey operations phase doing any remaining Science Validation Survey observations, such that DP2 could be realized as planned.
```
Comment (leannep) [inline timeline.tex:13]: phase **completing** any remaining Science Validation Survey observations, such that DP2 could be realized as planned.

**Review by leannep — CHANGES_REQUESTED** [review]
Minor changes

File: commissioning.tex (hunk 1 above)
Comment (drphilmarshall) [reply commissioning.tex:21]: Thanks, that's better. The community has already contributed to the menu, so I'll make sure that comes across.

File: timeline.tex (hunk 2 above)
Comment (drphilmarshall) [reply timeline.tex:13]: Good, thanks - adopting. BTW Did you know you can suggest changes directly, with that little page +/- icon? Makes GitHub more like Google docs, which is cool (-:

**Comment by drphilmarshall** [comment]
Addressed @leannep 's review comments, merging now so she can rebase and complete her own edits.
